1-WebApp-OIDC/1-4-Sovereign/README.md

```diff
@@ -79,28 +80,15 @@ There is one project in this sample. To register it, you can:
 
 > Note that unless the Web App calls a Web API no certificate or secret is needed.
 
-### Step 2: Download/ Clone this sample code or build the application using a template
-
-This sample was created from the dotnet core 2.2 [dotnet new mvc](https://docs.microsoft.com/dotnet/core/tools/dotnet-new?tabs=netcore2x) template with `SingleOrg` authentication, and then tweaked to let it support tokens for the Azure AD V2 endpoint. You can clone/download this repository or create the sample from the command line:
-
-#### Option 1: Download/ clone this sample
-
-You can clone this sample from your shell or command line:
-
-  ```console
-git clone https://github.com/Azure-Samples/microsoft-identity-platform-aspnetcore-webapp-tutorial webapp
-cd webapp
-cd "1-WebApp-OIDC\1-1-MyOrg"
-  ```
-
-> Given that the name of the sample is very long, and so are the name of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+### Step 3: Configure your App
 
-  In the **appsettings.json** file:
+#### Option 1:  In the **appsettings.json** file:
   
-  - replace the `ClientID` value with the *Application ID* from the application you registered in Application Registration portal on *Step 1*.
-  - replace the `TenantId` value with he *Tenant ID* where you registered your Application on *Step 1*.
+  - replace the `Instance` value with the relevant authority value for the National Cloud, your application is registered with. [List of authority of National Clouds](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud#azure-ad-authentication-endpoints)
```

Comment:
I would give an example (pre-suppose that this folder contains a sample for Arlington for instance)